packages/icons/src/svg/filter-16.svg

```diff
@@ -0,0 +1 @@
+<svg height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m10 10-1 1h-2l-1-1zm3-3-1 1h-8l-1-1zm2-3-1 1h-12l-1-1z" fill="#404b5a" fill-rule="evenodd"/></svg>
```

Comment:
The `fill` color is not set correctly, this is due to an error when exporting the source files. This icon will not build correctly if `fill: #010101` is not set, right?

Reply:
No worries, good catch! Yeah it looks for `#010101` when doing the search and replace.

Reply:
Fixed in ab026e1